swap/swap_test.go

```diff
@@ -1069,6 +1073,40 @@ func TestPeerProcessAndVerifyChequeInvalid(t *testing.T) {
 	}
 }
 
+func TestSwapLogToFile(t *testing.T) {
+
+	var err error
```

Comment:
you don't need this one

Reply:
TestSwapLogToFile test that swap is logging correctly to the dir.

```
TestResetBalance(t)
....

if !strings.Contains(s, "sending cheque") {
		t.Fatalf("expected the log to contain sending cheque")
	}
```

Reply:
sorry, i wasn't clear enough: i meant that you don't need this `var` declaration

Reply:
fixed, it was an appendage from refactored code.
Thanks for spotting that.